docs/Makefile

```diff
@@ -175,3 +175,12 @@ pseudoxml:
 	$(SPHINXBUILD) -b pseudoxml $(ALLSPHINXOPTS) $(BUILDDIR)/pseudoxml
 	@echo
 	@echo "Build finished. The pseudo-XML files are in $(BUILDDIR)/pseudoxml."
+
+makedocsmessages: gettext
+	cd docs && sphinx-intl update -p _build/locale -l en
+
+ensurecrowdinclient:
+	ls -l crowdin-cli.jar || wget https://crowdin.com/downloads/crowdin-cli.jar # make sure we have the official crowdin cli client
+
+uploaddocsmessages: ensurecrowdinclient
+	java -jar crowdin-cli.jar upload sources -b `git symbolic-ref HEAD | xargs basename`
```

Comment:
This is currently how we are doing it for the Kolibri interface strings also, but yeah, this could lead to unintended consequences.

More annoying is the 'downloadmessages' command where you have to be on the relevant release branch to get the messages.